config/routes.rb

```diff
@@ -1,3 +1,8 @@
 Rails.application.routes.draw do
+  devise_for :users
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+
+  root to: "home#index"
+
+  resources :users
```

Comment:
Why do you have `resources :users` when you already have devise handling `users` this causes an error and overwrites the user routes to be invalid since there is no user controller or associated views in your application. Remove this and your user routes should work fine. [REQUIRED]

Reply:
If you, of course, wanna change how the users routes work with devise you can read the documentation: https://github.com/plataformatec/devise.

For instance, it currently does not have users#index route, this may be something you want to add. 🤷‍♀